tensorflow-base-cpu-linwin/meta.yaml

```diff
@@ -123,7 +119,6 @@ requirements:
     - numpy >=1.20  # [not aarch64]
```

Comment:
We need `keras` instead of keras-preprocessing, see https://github.com/tensorflow/tensorflow/blob/2ea19cbb575d076b4f521d3603211c8316ad5f8f/tensorflow/tools/pip_package/setup.py#L103
I guess in host too